modules/php/Core/Notifications.php

```diff
@@ -331,6 +331,15 @@ public static function scribbleAstronaut(Player $player, Scribble $scribble)
     ]);
   }
 
+  public static function scribblePlanning(Player $player, Scribble $scribble)
+  {
+    $msg = clienttranslate('${player_name} crosses off a Planning symbol');
+    static::pnotify($player, 'addScribble', $msg, [
```

Comment:
You can use the generic addScribble method here, check how i refactored the others notifs

Reply:
Done